birdhouse/optional-components/wps-healthchecks/docker-compose-extra.yml

```diff
@@ -0,0 +1,80 @@
+version: "3.4"
+services:
+  catalog:
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "curl",
+          "--insecure",
+          "--fail",
+          "https://localhost${TWITCHER_PROTECTED_PATH}/catalog?service=WPS&request=GetCapabilities",
```

Comment:
Humm, why go through twitcher?  This will force anon access.  I assume a health check for each of the bird standalone.  If you want to check if anon access is enabled, I am not sure this is the right thing to do.

Reply:
Humm, does `localhost` even works?  `localhost` means the container running the bird, that container do not know about twitcher.

Reply:
I used the same definition that was defined on `thredds`. 
I haven't looked into it that much. You're probably right.

Reply:
fixed in fed17402df0fb43492715e018870a5abac92dc4b
tested each bird individually with `curl` / `python requests` according to what was available in containers